src/litserve/server.py

```diff
@@ -422,6 +424,8 @@ def __init__(
         stream: bool = False,
         spec: Optional[LitSpec] = None,
         max_payload_size=None,
+        middlewares: Optional[list[tuple[type[_MiddlewareClass[P]]], dict]] = None,
+        exc_handlers: Optional[list[tuple[int | type[Exception], ExceptionHandler]]] = None,
```

Comment:
please keep the scope minimal and only implement middlewares in this PR.

Reply:
ok, I removed the exception handlers part.